agr_java_core/src/main/java/org/alliancegenome/api/entity/GeneDiseaseAnnotationDocument.java

```diff
@@ -15,4 +19,8 @@ public GeneDiseaseAnnotationDocument() {
 		setCategory("gene_disease_annotation");
 	}
 
+	@Override
+	public String toString() {
```

Comment:
I would not override the toString

Reply:
This is helpful when debugging the code.

Reply:
Maybe but overriding the toString shouldn't be done control it via the lombok @ToString annotation on the class.